docs/discover/search.asciidoc

```diff
@@ -88,6 +88,67 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving and Loading Queries
+Saving queries enables you to reload them into <<discover, Discover>>, <<visualize, visualizations>> and <<dashboard, dashboards>>.
+Saving a query saves the query string, the filters and the time filter but, unlike searches, the index pattern is not saved, making saved queries portable.
+
+==== Read only access
+When you have insufficient privileges to save queries, the following indicator in Kibana will be displayed and the *Save* button won't be visible in the saved query management pop-over.
+
+[role="screenshot"]
+image::discover/images/saved-query-management-component-readonly.png["Example of the saved query management popover with read only access to Discover",width="80%"]
+
+==== Saving a Query
+To save the current query string, filters and time filter:
+
+. Click *#* in the search bar, next to the query string input.
+. Click the *Save* button in the popover.
+. Enter a name, a description and select the filter options you would like to include (filters are automatically included but the time filter is not) and click *Save*.
+
+[role="screenshot"]
+image::discover/images/saved-query-management-component.png["Example of the saved query management popover with a list of saved queries with write access",width="80%"]
+
+[role="screenshot"]
```

Comment:
Is this image needed?  If so,  suggest this:

3.  Enter a name and a description, and then select the filter options that you want to include. Filters are automatically included but the time filter is not.
image showing description field

4.  Click **Save**.
Image showing list of objects

Use a "+" between the step and the image to indent the image under the step.

Reply:
https://github.com/elastic/kibana/pull/43731/commits/b3163d0499ebb516e91879a34d4508e9e5deca57#diff-87a4343c7de08863ed7fa9d869b5c5f1R110